apps/src/templates/projects/FeaturedProjects.jsx

```diff
@@ -8,23 +8,30 @@ import {
 
 export default class FeaturedProjects extends React.Component {
   static propTypes = {
-    currentFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
+    activeFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
       .isRequired,
-    archivedUnfeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
+    archivedFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
+      .isRequired,
+    savedFeaturedProjects: PropTypes.arrayOf(featuredProjectDataPropType)
       .isRequired,
   };
 
   render() {
     return (
       <div>
-        <h3>Currently Featured Projects</h3>
+        <h3>Active Featured Projects (currently displayed in Gallery)</h3>
         <FeaturedProjectsTable
-          projectList={this.props.currentFeaturedProjects}
+          projectList={this.props.activeFeaturedProjects}
           tableVersion={featuredProjectTableTypes.current}
         />
+        <h3>Saved Featured Projects</h3>
+        <FeaturedProjectsTable
+          projectList={this.props.savedFeaturedProjects}
+          tableVersion={featuredProjectTableTypes.saved}
+        />
```

Comment:
@samantha-code - lmk what you think - thanks!

Reply:
I dont want to remove the projects in the current featured projects table as the basis for shortlisted/saved projects so I have a backlog to feature in the future. I agree that there are a ton of tables and it is a big tedious. I love the idea of a single table, but I would like the ability to filter the table on the "status" so that I could see all featured projects together etc. (which is what a single table gets us).

Reply:
Update: after chatting with @samantha-code , I'm going to refactor...
- Use a table named 'favorite_projects' instead of 'featured_projects'
- Three statuses: 'featured', 'never_featured', and 'previously_featured'
- Display in one table at 'projects/featured'
- Quick actions will be different depending on status (may hide/disable options that are not offered for a particular status)
- Display columns: 'Project Name', 'Type', 'Topic', 'Featured', 'Unfeatured', 'Quick Actions' ('Featured' and/or 'Unfeatured' could be blank depending on status of project)